ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewAppTest.java

```diff
@@ -0,0 +1,72 @@
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.regex.Pattern;
+
+public class NewAppTest extends BaseTest {
+
+    @Test
+    public void newApplication() {
+        int x = (int) (Math.random() * 1000);
+        driver.get("https://network-international--qa.cs83.my.salesforce.com/");
+        System.out.println("Open application (url) for testing");
+        Assert.assertEquals(driver.getTitle(), "Login | Salesforce", "Incorrect page (login page) is not opened");
+        System.out.println("Check that correct Login page is opened");
+        WebElement username = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 username']");
+        username.sendKeys("yauhen_valodzin@epam.com.qa");
+        System.out.println("Find and populate Username field");
+        WebElement password = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 password']");
+        password.sendKeys("Ceakt_1234");
+        System.out.println("Find and populate Password field");
+        WebElement button = driver.findElementByXPath("//input[@class='button r4 wide primary']");
+        button.click();
+        System.out.println("Find and click Login button");
+        WebElement dynamicElement = (new WebDriverWait(driver, 10).until(ExpectedConditions.presenceOfElementLocated((By.xpath("//span[text()='Recently Viewed']")))));
+        System.out.println("Add explicit wait");
+        Assert.assertEquals(driver.getTitle(), "Recently Viewed | Applications | Salesforce", "Incorrect Home page is not opened");
+        System.out.println("Check that correct Home page (after login) is opened");
+        WebElement appDropdown = driver.findElementByXPath("//span[text()='Applications']/../../one-app-nav-bar-item-dropdown//*[@role='button']");
+        appDropdown.click();
+        System.out.println("Find and click Applications dropdown");
+        WebElement createNewAppButton = driver.findElementByXPath("//span[text()='New Application']");
+        createNewAppButton.click();
+        System.out.println("Find and click New Application option");
+        WebElement formTradeName = driver.findElementById("input-3");
+        formTradeName.sendKeys("yauhen0707" + x);
+        System.out.println("Find and populate Trade Name field");
+        WebElement formEmail = driver.findElementByName("input-12");
+        formEmail.sendKeys("yauhen" + x + "@example.com");
+        System.out.println("Find and populate Merchant Email field");
+        WebElement formNumberOfAdditionalOutlets = driver.findElementById("input-01");
+        formNumberOfAdditionalOutlets.sendKeys("1");
+        System.out.println("Find and populate Number of additional outlets field");
+        WebElement formBusinessNature = driver.findElementByXPath("//div/select/option[@value='Computer Services']");
+        formBusinessNature.click();
```

Comment:
add some verification after click that action applied. And following actions also

Reply:
Done